apps/web/src/components/Shared/Markup/MarkupLink/ExternalLink.tsx

```diff
@@ -8,13 +8,14 @@ const ExternalLink: FC<MarkupLinkProps> = ({ href, title = href }) => {
     return null;
   }
 
+  let link = href;
   if (!href.includes('://')) {
```

Comment:
what if user just posts `://` in the post? imo also it looks like emoticon symbol too 😅